db/db.js

```diff
@@ -0,0 +1,5 @@
+const mongoose = require('mongoose');
```

Comment:
@seungprk Is this file still in your db folder? I can see it here but not when I look at the directories in your module.

Reply:
Yes, still there. If you go to main page of this repo and select branch and go inside the 'db' folder, you will be able to find.